wallet/src/descriptor/error.rs

```diff
@@ -68,7 +68,7 @@ impl fmt::Display for Error {
             ),
             Self::MultiPath => write!(
                 f,
-                "The descriptor contains multipath keys, which are not supported yet"
+                "The descriptor contains multipath keys with invalid number of paths (must have exactly 2 paths for receive and change)"
```

Comment:
Please also update the enum docs on line 24, should have more or less the same message as you have here.